src/drawing/DrawingCreator.tsx

```diff
@@ -163,6 +175,9 @@ export default function DrawingCreator({
                     </DrawingPathGroup>
                 </DrawingSVG>
             )}
+            {isHovering && (
+                <DrawingCursor className="ba-DrawingCreator-cursor" color={color} getGlobalPosition={getPosition} />
```

Comment:
Do we need to wrap `getPosition` in `React.useCallback` to avoid unnecessary rerenders?